### DISCUSSION TITLE
Welcome to FooDMe2 Discussions!

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gregdenay** [comment]
Hello @marchoeppner :wave:

**Comment by marchoeppner** [comment]
@gregdenay Hi - ich habe die Installation etwas umbauen müssen, damit die Ordnernamen noch Sinn machen. Das heisst für den aktuellen Code müsstest Du einmal die Referenzen neu installieren (--build_references --run_name build)
